Flow.Launcher.Test/FuzzyMatcherTest.cs

```diff
@@ -142,20 +144,20 @@ public void GivenQueryString_WhenAppliedPrecisionFiltering_ThenShouldReturnGreat
             var rawScore = matcher.FuzzyMatch(queryString, compareString).RawScore;
 
             // Should
-            Assert.AreEqual(expectedScore, rawScore, 
+            Assert.AreEqual(expectedScore, rawScore,
                 $"Expected score for compare string '{compareString}': {expectedScore}, Actual: {rawScore}");
         }
 
         [TestCase("goo", "Google Chrome", SearchPrecisionScore.Regular, true)]
         [TestCase("chr", "Google Chrome", SearchPrecisionScore.Low, true)]
         [TestCase("chr", "Chrome", SearchPrecisionScore.Regular, true)]
-        [TestCase("chr", "Help cure hope raise on mind entity Chrome", SearchPrecisionScore.Regular, false)]
```

Comment:
why is this test case removed?

Reply:
Because the acronym search will make this case matched, through `cure hope raise`. I think it is not the original idea for this test?

Reply:
This was to prevent fuzzy match bring up this kind of unrelated result. Maybe scoring needs adjusting so three acronyms shouldn't being up this long string?

Reply:
emmm agree with you.